### PR TITLE
Stop using LiveServerTestCase for webdriver and integration tests

### PR DESCRIPTION
**Pull request by hqpho**
LiveServerTestCase uses fork(), which causes intermittent issues on specific platforms. Instead, run NL server and website server separately. Instead use a separate script to start up one instance each of website and NL servers and have all tests use the same instances.

Other changes:
- Increase parallelization for all tests. Webdriver tests finish in <5 min now!
- Fix how one NL requirement is loaded (it was printing a warning before)
- Update node packages Docker image

### CODE REVIEW

File: run_servers.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+VERBOSE=false
+if [[ "$1" == "--verbose" ]]; then
+  VERBOSE=true
+fi
+
+exit_with=0
+
+function cleanup() {
+  pkill -P $$ || true
```
Comment (dwnoble) [inline run_servers.sh:32]: Add comment

File: run_servers.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  exit $exit_with
+}
+
+trap 'exit_with=$?; cleanup' EXIT
```
Comment (dwnoble) [inline run_servers.sh:36]: Add comment

File: run_test.sh [hunk 3]
```diff
@@ -25,6 +25,31 @@ function setup_python {
   deactivate
 }
 
+function start_servers() {
+  function cleanup() {
+    pkill -P $$ || true
```
Comment (dwnoble) [inline run_test.sh:30]: Add comment

File: run_test.sh [hunk 4]
```diff
@@ -25,6 +25,31 @@ function setup_python {
...
+    if [[ -n "$VIRTUAL_ENV" ]]; then
+      deactivate
+    fi
+    exit $exit_with
+  }
+  trap 'exit_with=$?; cleanup' EXIT
+  ./run_servers.sh &
+  SERVERS_PID=$!
```
Comment (dwnoble) [inline run_test.sh:38]: Add comment

File: run_test.sh [hunk 5]
```diff
@@ -25,6 +25,31 @@ function setup_python {
...
 
+function start_servers() {
+  function cleanup() {
+    pkill -P $$ || true
+    if [[ -n "$VIRTUAL_ENV" ]]; then
+      deactivate
+    fi
+    exit $exit_with
+  }
+  trap 'exit_with=$?; cleanup' EXIT
```
Comment (dwnoble) [inline run_test.sh:36]: Add comment

**Review by dwnoble — APPROVED** [review]
Thank you Hannah!!!

File: run_servers.sh [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+trap 'exit_with=0; cleanup' SIGINT SIGTERM
+
+if lsof -i :6060 > /dev/null 2>&1; then
+  echo "Port 6060 (for NL server) is already in use. Please stop the process using that port."
```
[the remark this answers is not in the document]
Comment (hqpho) [reply run_servers.sh:40]: Oh great call! I'll update ports and add a comment about why.

**Pull request merged**